src/store/people.js

```diff
@@ -2,6 +2,18 @@ const initialState = () => ({
   results: []
 })
 
+// Helper function to fetch links from the API and return name
+// of starship/planet/vehicle/etc.
+async function renderAttributeName (url) {
+  const results = await fetch(url)
```

Comment:
This is confusing because you're mixing 2 different but interchangeable concepts: promises and async/await.